src/stars.css

```diff
@@ -0,0 +1,42 @@
+.stars {
+  display: flex;
+  margin: 0;
+  padding-left: 0;
+  list-style-type: none;
+}
+
+.stars__star {
+  display: flex;
+  background: url(./images/star.svg) center no-repeat;
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars--0 :nth-child(-n + 0) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--1 :nth-child(-n + 1) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--2 :nth-child(-n + 2) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--3 :nth-child(-n + 3) {
```

Comment:
Avoid using tag names for styling. Instead of using ':nth-child', consider adding a specific class to the elements you want to style. This will improve performance and maintainability.